app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -117,6 +142,7 @@ private void fillDatabaseWithDemoData() {
             e.printStackTrace();
```

Comment:
That is a good one. But since it is not done anywhere yet I'll create a separate PR for that where I add a log for all instances of `.printStackTrace();` 👍